sunpy/util/tests/test_net.py

```diff
@@ -21,6 +21,7 @@ def test_slugify():
     assert sunpy.util.net.slugify("äb c", "b_c")
     assert sunpy.util.net.slugify("file.greg.fits") == "file_greg.fits"
     assert sunpy.util.net.slugify("file.greg.fits", "x") == "filexgreg.fits"
+    assert sunpy.util.net.slugify("file.name.fits.hdu") == "file_name.fits.hdu"
```

Comment:
Can you add the examples from the issue so we can double check?

Reply:
Sure i will add the example,